src/plugins/index_pattern_editor/public/components/index_pattern_editor_flyout_content.tsx

```diff
@@ -291,10 +277,6 @@ const IndexPatternEditorFlyoutContentComponent = ({
     [http, allowHidden, allSources, type, rollupIndicesCapabilities, searchClient, isLoadingSources]
   );
 
-  useEffect(() => {
-    reloadMatchedIndices(title);
```

Comment:
@sebelga pointed out that this might be redundant: https://github.com/elastic/kibana/pull/109500#discussion_r694666430
I tried to remove and indeed don't see any regressions

(not related to the main fix, but way too minor to extract to separate pr)